dev-libs/libunibreak/libunibreak-4.0.recipe

```diff
@@ -0,0 +1,81 @@
+SUMMARY="Line and word breaking library"
+DESCRIPTION="An implementation of the line breaking and word breaking \
+algorithms as described in Unicode Standard Annex 14 and Unicode Standard \
+Annex 29."
+HOMEPAGE="http://vimgadgets.sourceforge.net/libunibreak/"
+COPYRIGHT="2016 Andreas Röver"
+LICENSE="Zlib"
+REVISION="1"
+SOURCE_URI="https://github.com/adah1972/libunibreak/archive/libunibreak_4_0.tar.gz"
+CHECKSUM_SHA256="8ea270d2d9dea6b633f7de1a4a624163c3549b4b6f5031f08c6b96d704d58aa4"
+SOURCE_DIR="libunibreak-libunibreak_4_0"
```

Comment:
Suggestion:
```diff
-SOURCE_URI="https://github.com/adah1972/libunibreak/archive/libunibreak_4_0.tar.gz"
+SOURCE_URI="https://github.com/adah1972/libunibreak/archive/libunibreak_${portVersion//./_}.tar.gz"
 CHECKSUM_SHA256="8ea270d2d9dea6b633f7de1a4a624163c3549b4b6f5031f08c6b96d704d58aa4"
-SOURCE_DIR="libunibreak-libunibreak_4_0"
+SOURCE_DIR="libunibreak-libunibreak_${portVersion//./_}"
```
Well, maybe this is not a good idea. After all, if there are no more releases since 2016, then we can probably stick to the hard coded `4_0`.
(Feel free to choose whatever you think is best.)

Reply:
Upstream also provide an another tarball: https://github.com/adah1972/libunibreak/releases/download/libunibreak_4_0/libunibreak-4.0.tar.gz

This tarball has a different checksum and appears to contains docs as well, so you might want to use it instead.